Alpaca.Markets/Messages/JsonStreamAgg.cs

```diff
@@ -9,9 +9,6 @@ internal sealed class JsonStreamAgg : IStreamAgg
         [JsonProperty(PropertyName = "sym", Required = Required.Always)]
         public String Symbol { get; set; }
 
-        [JsonProperty(PropertyName = "x", Required = Required.Always)]
```

Comment:
Use `Default` mode for `Required` property - it will allow to de-serialize JSON even w/o this property.

Reply:
Re-added the property and set Required to `Default`.